alignak/daemons/brokerdaemon.py

```diff
@@ -420,17 +423,23 @@ def get_new_broks(self, i_type='scheduler'):
                 logger.exception(exp)
                 sys.exit(1)
 
-    def get_retention_data(self):
+    def get_retention_data(self):  # pragma: no cover, useful?
         """Get all broks
 
+        TODO: using retention in the arbiter is dangerous and
```

Comment:
Ok, the comment is not correct. but retention in the broker does not make any sense. No more than in the arbiter because data stored are message from the queues and broks to handle ... this is very dangerous to store and reload such data!